platforms/pf_test/include/motion_control.hpp

```diff
@@ -0,0 +1,131 @@
+    (1000 * platform_max_acc_m_per_s2 * motion_control_thread_period_ms * motion_control_thread_period_ms) \
+    / (1000 * 1000)
+    );          ///< Maximum linear acceleration (mm/<motion_control_thread_period_ms>²)
+constexpr double platform_max_speed_linear_mm_per_period = (
+    (1000 * platform_max_speed_m_per_s * motion_control_thread_period_ms) \
+    / 1000);    ///< Maximum linear speed (mm/<motion_control_thread_period_ms>)
+constexpr double platform_low_speed_linear_mm_per_period = (platform_max_speed_linear_mm_per_period / 4);     ///< Low angular speed (deg/<motion_control_thread_period_ms>)
+constexpr double platform_normal_speed_linear_mm_per_period = (platform_max_speed_linear_mm_per_period / 2);  ///< Normal angular speed (deg/<motion_control_thread_period_ms>)
+
+// Angular maximum speed and acceleration
+constexpr double platform_max_acc_deg_per_s2 = 360;  ///< Maximum acceleration (deg/s²)
+constexpr double platform_max_speed_deg_per_s = 720; ///< Maximum speed (deg/s)
+constexpr double platform_max_acc_angular_deg_per_period2 = (
+    (platform_max_acc_deg_per_s2 * motion_control_thread_period_ms * motion_control_thread_period_ms) \
+    / (1000 * 1000)
+    );  ///< Maximum angular acceleration (deg/<motion_control_thread_period_ms>)
+constexpr double platform_max_speed_angular_deg_per_period = (
+    (platform_max_speed_deg_per_s * motion_control_thread_period_ms) \
+    / 1000);    ///< Maximum angular speed (deg/<motion_control_thread_period_ms>)
+constexpr double platform_low_speed_angular_deg_per_period = (platform_max_speed_angular_deg_per_period / 4);       ///< Low angular speed (deg/<motion_control_thread_period_ms>)
+constexpr double platform_normal_speed_angular_deg_per_period = (platform_max_speed_angular_deg_per_period / 2);    ///< Normal angular speed (deg/<motion_control_thread_period_ms>)
+/// @}
+
+/// Get pose to reach from protobuf message
+void pf_handle_target_pose(cogip::uartpb::ReadBuffer &buffer);
+
+/// Get start pose from protobuf message
+void pf_handle_start_pose(cogip::uartpb::ReadBuffer &buffer);
+
+/// Initialize motion control
+void pf_init_motion_control(void);
+
+/// Start motion control
+void pf_start_motion_control(void);
+
+/// Send current robot pose in Protobuf format over UART
+void pf_send_pb_pose(void);
+
+/// Send current robot state in Protobuf format over UART
+void pf_send_pb_state(void);
+
+/// Print robot state
+void pf_print_state(void);
+
+/// Compute current speed and pose
+void compute_current_speed_and_pose(
+    cogip::cogip_defs::Polar &current_speed,    ///< [out]  robot current speed
+    cogip::cogip_defs::Pose &current_pose       ///< [out]  robot current pose
+    );
+
+
+/// Apply the given command to the motors
+void pf_motor_drive(
+    const cogip::cogip_defs::Polar &command     ///< [in]   linear and angular speeds command
+    );
+
+} // namespace actuators
+
+} // namespace pf
+
+} // namespace cogip
```

Comment:
Missing empty line at EOF.

Reply:
same weird, got them but in a further commit maybe